packages/types/src/clerk.ts

```diff
@@ -940,6 +940,10 @@ export type SignInProps = RoutingOptions & {
    * Additional arbitrary metadata to be stored alongside the User object
    */
   unsafeMetadata?: SignUpUnsafeMetadata;
+  /**
+   * Enable the combined sign-in-or-up flow for `<SignIn />` component instance.
+   */
+  combinedFlow?: boolean;
```

Comment:
I don't _love_ this name for the public API 🤔

Spitballing:
* `combined`
* `allowSignUp`

Reply:
- `orSignUp`
- `withSignup`
- `includeSignUp`